.github/workflows/backend.yml

```
@@ -79,7 +79,20 @@ jobs:
         run: cargo clippy
         env:
           SQLX_OFFLINE: "true"
-  
+
+  formatting:
+    name: cargo fmt
+    runs-on: ubuntu-latest
+    defaults:
+        run:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)